pyproject.toml

```diff
@@ -13,4 +13,9 @@ requires-python = ">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*"
 requires = [
   "configparser >= 3.5; python_version < '3'",
   "importlib-resources; python_version < '3.7'",
+  "build >= 0.2.0",  # not a hard runtime requirement -- we can softfail
+  "packaging",  # not a hard runtime requirement -- we can softfail
 ]
+
+[tool.flit.scripts]
+python-installer = "installer.__main__:entrypoint"
```

Comment:
Yeah, it would also be good to have cli flags for settings stuff like the installation `--root` similar to setuptools, for example our setuptools target build/installs have to [set a bunch of stuff](https://github.com/buildroot/buildroot/blob/master/package/pkg-python.mk) so that things are built against the right interpreter and installed in the right place:
```
# Target setuptools-based packages
PKG_PYTHON_SETUPTOOLS_ENV = \
	_PYTHON_SYSCONFIGDATA_NAME="$(PKG_PYTHON_SYSCONFIGDATA_NAME)" \
	PATH=$(BR_PATH) \
	$(TARGET_CONFIGURE_OPTS) \
	PYTHONPATH="$(if $(BR2_PACKAGE_PYTHON3),$(PYTHON3_PATH),$(PYTHON_PATH))" \
	PYTHONNOUSERSITE=1 \
	_python_sysroot=$(STAGING_DIR) \
	_python_prefix=/usr \
	_python_exec_prefix=/usr

PKG_PYTHON_SETUPTOOLS_INSTALL_TARGET_OPTS = \
	--prefix=/usr \
	--executable=/usr/bin/python \
	--single-version-externally-managed \
	--root=$(TARGET_DIR)
```

Reply:
Please see #58.